### PR TITLE
[CLI-62][CLI-65] fix bad request on submission

### PR DESCRIPTION
**Pull request by ericohwotu**
- handled lower case eori's for routing
- city and postcode in the correct fields for pincipal
- country code used from user answers

### CODE REVIEW

**Comment by ericohwotu** [comment]
test this please

File: app/services/DeclarationRequestService.scala [hunk 1]
```diff
@@ -441,26 +424,15 @@ class DeclarationRequestService @Inject() (
 
   // TODO: Improve by changing ConsignorDetails to have a Consignor Address instead
```
Comment (amaalali) [inline app/services/DeclarationRequestService.scala:425]: I think we can remove this comment now

```suggestion
```

File: app/services/DeclarationRequestService.scala [hunk 2]
```diff
@@ -441,26 +424,15 @@ class DeclarationRequestService @Inject() (
...
-      .getOption(address)
-      .map {
-        case CommonAddress(addressLine1, addressLine2, addressLine3, country) =>
-          TraderConsignor(name, addressLine1, addressLine3, addressLine2, country.code.code, eori.map(_.value))
-      }
-      .get
+    TraderConsignor(name, addressLine1, postCode, addressLine2, country.code.code, eori.map(_.value))
   }
 
   // TODO: Improve by changing ConsigneeDetails to have a Consignee Address instead
```
Comment (amaalali) [inline app/services/DeclarationRequestService.scala:432]: I think we can remove this comment now

```suggestion
```

File: test/services/DeclarationRequestServiceSpec.scala [hunk 3]
```diff
@@ -494,6 +499,34 @@ class DeclarationRequestServiceSpec
         }
       }
 
+      "principalTraderDetails" - {
+        "has the postcode and city in the right field and the country is set to users answer" in {
+          val userAnswers   = arb[UserAnswers].sample.value
+          val journeyDomain = arbitraryNormalJourneyDomain.sample.value
```
Comment (amaalali) [inline test/services/DeclarationRequestServiceSpec.scala:505]: @MatthewAlexanderBeck  FYI

**Review by amaalali — APPROVED** [review]
Looks good 👍  There are two TODO comments that can now be removed from the code because of this improvement

**Pull request merged**